stubs/Flask-SQLAlchemy/flask_sqlalchemy/__init__.pyi

```diff
@@ -0,0 +1,84 @@
+from typing import Any
+
+from . import utils as utils
+from .model import DefaultMeta as DefaultMeta, Model as Model
+
+models_committed: Any
+before_models_committed: Any
+
+class SignallingSession:
```

Comment:
This inherits from SessionBase. We probably don't have stubs for that, so we should inherit from Any.

Reply:
Thanks! For this and the other missing bases, could you add a comment saying what the real base class is? That way, we can update it when we do have those classes in typeshed.